integrationos-archiver/src/config.rs

```diff
@@ -0,0 +1,39 @@
+use envconfig::Envconfig;
+use integrationos_domain::database::DatabaseConfig;
+use std::fmt::{Display, Formatter};
+
+#[derive(Envconfig, Clone)]
+pub struct SnapshotConfig {
```

Comment:
Is there a better name for this than snapshot?

The naming suggests it's a point-in-time snapshot of all the data, but it's actually a partial data dump from a specific time window.